contracts/RestrictedNORI.sol

```diff
@@ -829,52 +826,52 @@ contract RestrictedNORI is
     uint256 removalId,
     uint256 amount
   ) internal {
-    // slither-disable-next-line calls-loop
+    // slither-disable-next-line calls-loop choose to get the project id from the removal contract
```

Comment:
this would also be significantly approved with a fix like [this](https://github.com/nori-dot-eco/contracts/pull/249/files/998a7c2f78de2bf6a1f41f2c5161c04e02449ab5..fb97bb8a727a24cdc034f908b27899c6a7b61e26#r906863991) :(

Reply:
great there's a todo on the removal utils file now